include/xlnt/workbook/workbook.hpp

```diff
@@ -386,6 +386,8 @@ class XLNT_API workbook
 
     /// <summary>
     /// Returns the value of the given core property.
+    /// Assumes that the specified core_property exists (please call has_core_property() to check).
```

Comment:
Should we make the distinction between a non-existing property and a null variant?

Reply:
> Hmm, do you have a specific use case in mind?

Just avoiding the need for a try/catch or has_core_property call to obtain f.ex. the creator of a document.

Looking at the specs, it seems to me that for the core and extended properties a null property / variant (i.e. `vt:null`) is not allowed (most of them are not variant types anyway).

Even for custom properties, I guess returning a null variant would be fine for non-existing properties. 99,99% of the xlnt users is probably not interested in the difference. If one really wants to make the distinction, he can always call `has_custom_property` to find out.